src/reverse.jl

```diff
@@ -4,7 +4,6 @@ struct ReverseIndices{I, Inds <: AbstractIndices{I}} <: AbstractIndices{I}
 end
 
 Base.parent(inds::ReverseIndices) = getfield(inds, :inds)
-@propagate_inbounds Base.in(inds::ReverseIndices{I}, i::I) where {I} = in(parent(inds), i)
```

Comment:
Can we instead change the line to this?

```julia
@propagate_inbounds Base.in(inds::ReverseIndices) = in(parent(inds), i)
```

Reply:
I'll try this out and let you know. Is it possible you are missing an argument in the function signature though?